_layouts/CSGLabs.html

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
There's a much easier, Jekyll way to do this. Here's an example: https://github.com/jekyll/example/blob/gh-pages/about.md